Tests/Filter/DateRangeFilterTest.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Patrick Landolt <patrick.landolt@artack.ch>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Filter;
+
+use Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQuery;
+use Sonata\DoctrineORMAdminBundle\Filter\DateRangeFilter;
+
```

Comment:
Your name should be here : 

/**
 * @author Patrick Landolt <patrick.landolt@artack.ch>
 */